<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rankingWidget</class>
 <widget class="QWidget" name="rankingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>400</width>
     <height>498</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;方正舒体&quot;;
gridline-color: rgb(85, 170, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_before">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>640</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>640</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
